docs/feature-gates/permanent-and-stale-gates.md

```diff
@@ -7,38 +7,44 @@ slug: /feature-gates/permanent-and-stale-gates
+    - There are two ways to mark a gate as **Permanent**:
+      - When creating the gate: the **Permanent** box in the gate creation flow
+        ![image](https://user-images.githubusercontent.com/120431069/224768058-1a1b74a2-6b5d-4bfd-b73c-e2fc1f4a7a7f.png)
+      - After a gate has been created: click on the "..." menu and then "Mark Gate Permanent"
+        ![image](https://user-images.githubusercontent.com/120431069/224763304-2002e482-8ef0-4025-b13c-acb92ffb2bcc.png)
+  - Implications of marking a gate as **Permanent**
+    - No change in the gate’s behavior when called
+    - Easy filtering on feature gates catalog
+    - More caution in its management: while you are able to archive or delete a **permanent** feature gate, there will be a warning shown before proceeding
+    - Statsig will forgo reminding you to clean up these gates and may display them differently in the console
+  - You will be able to change the gate back to **Temporary** at any point.
+    - All newly created feature gates are marked as **Temporary**, unless marked otherwise (i.e. Permanent). Therefore, Statsig will not display the phrase **Temporary** in the feature gates Catalog or within the individual gates page.
 - **Stale Gates** (set by Statsig)
-    - **Stale feature gates** indicate to your team that these gates could be a good candidate for cleanup. Statsig automatically marks gates as stale based on the following definition (excludes Permanent and newly created gates) -
-        - 0 checks within last 30 days OR
-        - gate rolled out to 100% or 0%
-    - Implications of gates being marked as **Stale**
-        - No change in the gate’s behavior when called
-        - Easy filtering on Feature Gates catalog
-        - Will include the gate in Statsig’s nudges for cleanup (more below)
+  - **Stale feature gates** indicate to your team that these gates could be a good candidate for cleanup. Statsig automatically marks gates as stale based on the following definition (excludes Permanent and newly created gates) -
+    - 0 checks within last 30 days OR
+    - gate rolled out to 100% or 0%
+  - Implications of gates being marked as **Stale**
+    - No change in the gate’s behavior when called
+    - Easy filtering on Feature Gates catalog
+    - Will include the gate in Statsig’s nudges for cleanup (more below)
+  - **Stale Reasons** are the reason why a gate has been set as stale.
+    - **None** No Stale Gates should have their reason set as None, this is exclusively for **Temporary** or **Permanent** gates.
+    - **STALE_PROBABLY_DEAD_CHECK** There have been no checks in the last 30 days.
+    - **STALE_PROBABLY_LAUNCHED** The Gate is disabled and has likely launched.
+    - **STALE_PROBABLY_UNLAUNCHED** The Gate is disabled and has no default value.
+    - **STALE_NO_RULES** The Gate has no set rules.
```

Comment:
Pointing people to the console API as a way to query this information since these details aren't surfaced on the console itself. Or just put it on the console Docs page directly

Reply:
Added link to Console API off of the stale gate info